features/levering-koopsom.feature

```diff
@@ -0,0 +1,152 @@
+Functionaliteit: levering koopsom
```

Comment:
er bestond al een feature "levering koopsom.feature". We hebben dus twee bijna gelijknamige features. Kan je niet beter onderstaande toevoegen in het bestaande feature?